flytekit/contrib/notebook/helper.py

```diff
@@ -1,10 +1,10 @@
 import os as _os
 
 import six as _six
-from pyspark import SparkConf, SparkContext
 
 from flytekit.common.types.helpers import pack_python_std_map_to_literal_map as _packer
 from flytekit.contrib.notebook.supported_types import notebook_types_map as _notebook_types_map
+from flytekit.plugins import pyspark as _pyspark
```

Comment:
this will work because only when we access pyspark.`x` will it load the real module?

Reply:
Yeah, that's my understanding of lazy_loading here: https://github.com/lyft/flytekit/blob/master/flytekit/plugins/__init__.py#L5